docs/l10n.rst

```diff
@@ -0,0 +1,148 @@
+
+Both of these projects use `Pontoon`_ for translations.
+
+Addons-server
++++++++++++++
+
+Addons-server is the current front-end for AMO, it is also the API
+backend and home to the developer hub and reviewer tools. It is written in
+Python using the Django Framework.
+
+Testing changes to addons-server localizations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The strings will be available on -dev or stage depending on when they were
+committed see `When can I expect to see my strings land?`_ for details.
+
+The relevant hosts for addons-server are as follows:
+
+    * dev https://addons-dev.allizom.org/
+    * stage https://addons.allizom.org/
+    * prod https://addons.mozilla.org/
+
+Addons-frontend
++++++++++++++++
+
+Addons-frontend is the new frontend for AMO, it will eventually replace the
+addons-server frontend for user-pages. It is written in JavaScript and uses
+React at it's core. Addons-frontend currently provides an amo app (amo.po).
+
+Testing amo localizations
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The strings will be available on -dev or stage depending on when they were
+committed see `When can I expect to see my strings land?`_ for details.
+
+The hosts for the new frontend are the same as addons-server except be routed
+to the new frontend you either need:
+
+    * A "mamo" cookie with the value "on". You can set this by clicking
+      "View mobile site" in the footer.
+    * A Mobile user-agent string.
+
+Discovery-Pane app
+++++++++++++++++++
+
+The addons-frontend repo is also home to the Discovery pane app (disco.po)
+which is the site containing curated addons available at about:addons
+(tools -> Add-ons -> Get Add-ons).
+
+Whilst this looks like it's part of firefox it's actually a webpage served
+from ``https://discovery.addons.mozilla.org``.
+
+Discovery Pane editorial content
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The content for the disco pane comes from the API so if you want to
+translate that you will need to look in addons-server.
+
+Testing Discovery Pane localizations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The strings will be available on -dev or stage depending on when they were
+committed see `When can I expect to see my strings land?`_ for details.
+
+The hosts for the discovery pane are as follows:
+
+    * dev ``https://discovery.addons-dev.allizom.org``
+    * stage ``https://discovery.addons.allizom.org``
+    * prod ``https://discovery.addons.mozilla.org``
+
+Advanced Configuration for testing addons-frontend strings
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+This will be enough for basic string checks. If you need to test strings
+related to the install buttons you will need to enabled preferences to allow
+add-on installation to work on the development and stage servers.
+
+To fully configure Firefox to install add-ons on either dev/stage you will
+need to additionally do the following:
+
+    * Use nightly or dev edition
```

Comment:
It'd say "Use Nightly or Firefox Developer Edition"

We throw around "dev" a fair bit as an "edition" of the AMO site so clarity is better here. Plus: Branding™